.github/workflows/close-stale-issues.yml

```
@@ -0,0 +1,22 @@
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
```

actionlint (1.7.7)

the runner of "actions/stale@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)